README.md

```diff
@@ -101,9 +82,9 @@ bpftrace has several plugins/definitions, integrating the syntax into your edito
 
 ## License
 
-Copyright 2019 Alastair Robertson
+bpftrace is a [registered trademark](https://trademarks.ipo.gov.uk/ipo-tmcase/page/Results/1/UK00003916706) of Alastair Robertson
```

Comment:
+1
But I would skip the link, since bad actors might start timing attack close to expiration date.